src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -1,6 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Authors>Confluent Inc.;Andreas Heider</Authors>
+    <Description>Confluent's .NET Client for Apache Kafka</Description>
+    <Copyright>Copyright 2016-2017 Confluent Inc., Andreas Heider</Copyright>
+    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
```

Comment:
yep - i'll make some changes based on the above. thanks!

For release notes, I think we can add these with each PR and reset after a release. It's not a big deal if we forget / don't do this on each PR - the main thing is to make sure they are correct before tagging the repo with the release tag. The process is not ideal - there is potential to forget to do this, but I'm not sure what is better.